docs/semconv/database.md

```diff
@@ -78,24 +81,36 @@ All metrics in `db.instance` instruments should be attached to a [Instance resou
-| `db.tablespace.utilization` | The used percentage of the tablespace.         | 1    | Gauge            | Double     | Optional          | `tablespace_name`(Required) identifier. | {tablespace_name} |
-| `db.tablespace.max`         | The max size (in bytes) of the tablespace.     | By   | UpDownCounter    | Int64      | Optional          | `tablespace_name`(Required) identifier. | {tablespace_name} |
-
+| Name                        | Description                                 | Unit | Instrument Type  | Value Type | Requirement Level | Attribute Key(s)                        | Attribute Values  |
+|-----------------------------|---------------------------------------------|------|------------------|------------|-------------------|-----------------------------------------|-------------------|
+| `db.disk.usage`             | The size (in bytes) of the used disk space. | By   | UpDownCounter    | Int64      | Recommended       | `path`(Recommended) The disk path.      | (path)            |
+| `db.disk.utilization`       | The percentage of used disk space.          | 1    | Gauge            | Double     | Recommended       | `path`(Recommended) The disk path.      | (path)            |
+| `db.cpu.utilization`        | The percentage of used CPU.                 | 1    | Gauge            | Double     | Recommended       |                                         |                   |
+| `db.mem.utilization`        | The percentage of used memory               | 1    | Gauge            | Double     | Recommended       |                                         |                   |
+| `db.tablespace.size`        | The size (in bytes) of the tablespace.      | By   | UpDownCounter    | Int64      | Optional          | `tablespace_name`(Required) identifier. | {tablespace_name} |
+| `db.tablespace.used`        | The used size (in bytes) of the tablespace. | By   | UpDownCounter    | Int64      | Optional          | `tablespace_name`(Required) identifier. | {tablespace_name} |
+| `db.tablespace.utilization` | The used percentage of the tablespace.      | 1    | Gauge            | Double     | Optional          | `tablespace_name`(Required) identifier. | {tablespace_name} |
+| `db.tablespace.max`         | The max size (in bytes) of the tablespace.  | By   | UpDownCounter    | Int64      | Optional          | `tablespace_name`(Required) identifier. | {tablespace_name} |
+| `db.disk.writes`            | Actual number of writes to disk             | By   | UpDownCounter    | Int64      | Optional          |                                         |                   |
+| `db.disk.reads`             | Actual number of reads to disk              | By   | UpDownCounter    | Int64      | Optional          |                                         |                   |
```

Comment:
Same to above.

Reply:
It is the count that we fetching not the volume. Another option can be 
`Actual number of physical reads to disk`
Let me know your thought.

Reply:
Updated as per our discussion.